form.yml.erb

```diff
@@ -179,7 +198,32 @@ attributes:
           data-min-ppn-pitzer: 1,
           data-max-ppn-pitzer: 48,
           data-option-for-owens: true,
-          data-option-for-pitzer: true
+          data-option-for-pitzer: true,
+          data-option-for-kubernetes: false
+        ]
+      - [
+          "pitzer",   "pitzer",
+          data-option-for-owens: false,
+          data-option-for-pitzer: false,
+          data-option-for-kubernetes: true
+        ]
```

Comment:
Isn't this the same as the "48 core" option?

Reply:
Not necessarily, technically the worker CPU nodes might have 40 cores, not 48.  Also the core count has no impact on scheduling like it does in SLURM where core count node type can drastically change how things are scheduled.